portfolio/src/main/webapp/script.js

```diff
@@ -48,15 +48,22 @@ addRandomCat = () => {
   else catContainer.appendChild(catimg);
 }
 
+/*
+ * Use Potion API to create page content for blog post 1 from a Notion document.
+ */
 window.onload = () => {
   fetch("https://potion-api.now.sh/html?id=" + POST_ID)
     .then(res => res.text())
     .then(text => { document.getElementById("post1").innerHTML = text; 
   });
 }
 
-getServerMessage = () => {
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementById('message-container').innerText = quote;
+/* 
+ * Get message from server and insert it on the home page.
+ */
+window.onload = () => {
+  fetch('/data').then(response => {
+    // response.text() is the server message.
+    document.getElementById('message-container').innerText = response.text();
```

Comment:
Cleaner to have a single `window.onload` in the page, and have the different fetch calls inside it

```
window.onload = () => {
   // Fetch from Potion API to a, b, c.
   functionCallToGetPotionBlog();

  // Fetch from message server greeting, etc, etc.
  functionCallToGetMessageServer();
}
```

For the actual function calls, depending on how complicated the code becomes, could leave as is, or define them as separate methods using `async` javascript functions (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) since I believe `fetch` calls returns a promise.

there's some examples here: https://www.javascripttutorial.net/javascript-fetch-api/, copy and pasting one of the examples below:

```
async function fetchText() {
    let response = await fetch('/readme.txt');
    let data = await response.text();
    console.log(data);
}
```

Once we get to unit-testing code, having the fn's clearly separated and called-out will make it a lot easier to test independently, imo.